### PR TITLE
[Edit] CSS [Edit] CSS Borders: border-color

### PR DESCRIPTION
**Pull request by yeipkwa**
### Description

- Added css-border-color-example1.png and css-border-color-example2.png images media directory
- Added links to example images to border-color.md file.
- Closes Issue #3640 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/borders/terms/border-color/border-color.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ h1 {
   border-style: dotted;
 }
 ```
+![css-border-color-example1](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-border-color-example1.png)
```
Comment (ishg-153) [inline content/css/concepts/borders/terms/border-color/border-color.md:45]: Add more descriptive `alt-text` (text inside the square brackets).

It should be information about the output image.

File: content/css/concepts/borders/terms/border-color/border-color.md [hunk 2]
```diff
@@ -52,3 +53,4 @@ h1 {
   border-color: coral beige;
 }
 ```
+![css-border-color-example2](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-border-color-example2.png)
```
Comment (ishg-153) [inline content/css/concepts/borders/terms/border-color/border-color.md:56]: Same here, make the `alt-text` more descriptive.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@yeipkwa The output images look great! I have requested 2 changes regarding `alt-text`. Please make those.

**Review by yeipkwa — COMMENTED** [review]
Hi, I added more descriptive alt texts to the images. Let me know if there's anything else I need to change. Thanks!

**Review by ishg-153 — APPROVED** [review]
@yeipkwa Great! This looks ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@yeipkwa Looks good.

**Pull request merged**